### DISCUSSION TITLE
Unit tests for rest apis

### DISCUSSION THREAD

**Discussion by vamsibvv** [rahulvemula/SE-leasingPortal#87] [Show and tell]
Hi Team,

Using this area to see the progress of the unit tests written so far.

Completed unit tests:

1. GetUser
2. GetUserByEmail
3. GetLease
4. GetLeaseById

Thank you!!

**Comment by mitulmandaliya** [comment]
Thanks Vamsi. I'll write the test cases for remaining 2 apis by tomorrow EOD
